cpp/CMakeLists.txt

```diff
@@ -0,0 +1,140 @@
+cmake_minimum_required(VERSION 3.10.0)
+
+set(PROJECT_NAME Mikado-TestBuilders-Workshop)
+
+set(CMAKE_TOOLCHAIN_FILE ${CMAKE_CURRENT_SOURCE_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake
+  CACHE STRING "Vcpkg toolchain file")
+
+project(${PROJECT_NAME} CXX)
+
+enable_testing()
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+if (NOT TARGET gtest)
```

Comment:
Why do you use both vcpkg to get gtest and direct dowload here ?

Reply:
That is actually one of the points that I wasn't sure about! 
After a discussion with Antoine last week, I noticed that this was an unneeded step. 
I should be removing this next.